smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -457,6 +457,11 @@ pub struct ZeroSignatureThreshold;
 #[repr(transparent)]
 pub struct PublicKeyEd25519(pub [u8; 32]);
 
+impl PublicKeyEd25519 {
+    /// Create a new [`PublicKeyEd25519`] from bytes.
+    pub fn from_bytes(bytes: [u8; 32]) -> Self { PublicKeyEd25519(bytes) }
+}
```

Comment:
Why do we need this method?

I'd prefer not to have it since we have public fields already and this method does not verification compared to a similar method for the ed25519::PublicKey.